src/utils/holidays.js

```diff
@@ -0,0 +1,14 @@
+const emojis = new Map([
+  [`New Year's Day`, ":baby-new-year:"],
+  ["Birthday of Martin Luther King, Jr.", ":mlk:"],
+  [`Washington's Birthday`, ":george:"],
```

Comment:
Ditto, double quotes for consistency?

```suggestion
  ["Washington's Birthday", ":george:"],
```